source/Engine.cpp

```diff
@@ -2745,21 +2748,26 @@ void Engine::CreateStatusOverlays()
 		if(it->IsDestroyed())
 			continue;
 
+		static auto FLAGSHIP = Preferences::OverlayType::FLAGSHIP;
+		static auto FRIENDLY = Preferences::OverlayType::ESCORT;
+		static auto HOSTILE = Preferences::OverlayType::ENEMY;
+		static auto NEUTRAL = Preferences::OverlayType::NEUTRAL;
```

Comment:
Does something like this work?
```suggestion
		using enum Preferences::OverlayType;
```

Reply:
clang says yes

Reply:
what compiler is it?

Reply:
GCC 10.3, apparently.
For reference:
CI job | GCC version
-- | --
Ubuntu 22.04 GL | 11.4
Ubuntu 24.04 GL | 13.2
Windows | 12.2

MacOS uses AppleClang 15 and Windows Clang uses Clang 18.1.8, but those aren't directly as comparable.

Reply:
`using enum` is indeed not supported in gcc versions less than 11.
Near the bottom of the C++20 section: https://gcc.gnu.org/projects/cxx-status.html
https://wg21.link/p1099r5